asserts/asserts.go

```diff
@@ -58,6 +58,7 @@ func (at *AssertionType) MaxSupportedFormat() int {
 var (
 	AccountType         = &AssertionType{"account", []string{"account-id"}, assembleAccount, 0}
 	AccountKeyType      = &AssertionType{"account-key", []string{"public-key-sha3-384"}, assembleAccountKey, 0}
+	RepairType          = &AssertionType{"repair", []string{"repair-id"}, assembleRepair, 0}
```

Comment:
we need to decide what's really the primary key/how we will use it, especially in light of the idea of opening this further, that's the thing that is hardest to change after the fact